### DISCUSSION TITLE
Dependency from disabled module

### DISCUSSION THREAD

**Discussion by msquare98** [OrchardCMS/OrchardCore#11202] [General]
For example, i have a built a module "test1" and there is disabled module named "oldtest". I want data from "oldtest" for "test1". Is it possible to access the services or data from disabled module "oldtest" without enabling it. What i mean is, without Injecting service from disabled module, can i access the services the module offers? I need this because of old systems where "oldtest" is not enabled, i am building an app for all which should work for everyone but unfortunately i need services from "oldtest". Is there a workaround? any help is appreciated. Thank You

**Comment by ns8482e** [comment]
- Remove dependency of "oldtest" from Manifest of "test"
- Add project reference of "oldtest" to "test" and register services of "oldtest" directly on Startup of "test"
